build/integrationtests/Dockerfile

```diff
@@ -1,23 +1,29 @@
 ARG cycle
 ARG hub
+ARG integrationtests
 
 FROM ${hub}/deploy-env:${cycle}
-
 LABEL maintainer Rob Bovill <rbovill@lsst.org>
+LABEL integrationtests=${integrationtests}
```

Comment:
Consider adding this LABEL at the end of the build, or at least after you install the package.. This helps docker build reuse layers. 

You can see how we do on the other build, e.g.:

https://github.com/lsst-ts/ts_cycle_build/blob/c318fd5efcb6fb51d829ee29886b18f5a3b8271a/build/ataos/Dockerfile#L55-L58

Reply:
Done